assets/css/meta-box-quiz-editor.scss

```diff
@@ -192,6 +192,9 @@
 							color: red;
 						}
 					}
+					&.question_delete--without-edit {
```

Comment:
This block should be inside the `.question_table_delete, .question_multiple_delete` block.